trigger/build.go

```diff
@@ -123,10 +125,19 @@ func (t *buildTriggers) Fetch(e *event.EventLogEntry) error {
 		return errors.New("couldn't find build")
 	}
 
-	tasks, err := task.FindAll(task.ByBuildId(t.build.Id))
-	if err != nil {
-		return errors.Wrap(err, "failed to fetch tasks")
+	var tasks []task.Task
+	if e.EventType == event.BuildGithubCheckFinished {
```

Comment:
Do you need to have this split here? It doesn't look like the trigger you added does anything with the list of tasks

Reply:
These tasks are used by [TaskStatusToDesc](https://github.com/evergreen-ci/evergreen/blob/49c7a2b8b30084c7973337bbb11986e3dac86756/trigger/build.go#L25). Without filtering tasks, the descriptions will wrongfully include success/failures of tasks outside of the aliases.